src/components/messenger/list/index.tsx

```diff
@@ -95,7 +96,7 @@ export class Container extends React.Component<Properties, State> {
       isFirstTimeLogin: registration.isFirstTimeLogin,
       isInviteNotificationOpen: registration.isInviteToastOpen,
       userName: user?.data?.profileSummary?.firstName || '',
-      userHandle: (hasWallet ? user?.data?.wallets[0]?.publicAddress : user?.data?.profileSummary?.primaryEmail) || '',
```

Comment:
shouldn't this be featureFlagged (via `allowEmailRegistration`)? We're assuming everyone logs in via wallet but i think it is still a feature flagged feature

Reply:
`userHandle` no longer uses `primaryEmail` here so we shouldn't need to feature flag this. If the user has a `primaryZID`, we'll render that, if not, we'll render the wallet address, if that's not available for whatever reason then we won't render the `userHandle`

Reply:
We still have a featureFlag `allowEmailRegistration`. What happens when i enable that & register via email & I don't have a ZID? That's what i was asking.